stage6/07-aditof-sdk/00-run.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+ADITOF_SDK_BRANCH=master # this branch will be bumped for new releases
```

Comment:
is the a release branch now ? If something will break in the  TOF master it will break the entire Kuiper build

Reply:
The branch will be updated, when the new TOF release will be out. We can merge it afterwards, Is there a problem with this approach? Also the current release 1.6.0 has a setup script that does some `apt-get install` in it and is not really updated for usage in kuiper build. https://github.com/analogdevicesinc/aditof_sdk/pull/459 This changes are made for the Kuiper build of the TOF sdk and will only be available with release 2.0.0.

Reply:
then should we push this to when we have the release for TOF?